src/nvim/eval.c

```diff
@@ -9106,6 +9107,27 @@ void set_vim_var_dict(const VimVarIndex idx, dict_T *const val)
   }
 }
 
+/*
+ * Set the v:argv list.
+ */
+  void
```

Comment:
remove the space and make sure the linter passes: the comments we use are  `///`